addons/compat_ace_overpressure/functions/fnc_firedEHBB.sqf

```diff
@@ -0,0 +1,106 @@
+#include "..\script_component.hpp"
+/*
+ * Author: joko // Jonas
+ * Handle fire of local launchers. Called from the unified fired EH only for the local player.
+ *
+ * Arguments:
+ * None. Parameters inherited from EFUNC(common,firedEH)
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [player, "launch_RPG32_F", "launch_RPG32_F", "Single", "R_PG32V_F", "RPG32_F", projectile] call ace_overpressure_fnc_firedEHBB;
+ *
+ * Public: No
+ */
+
+//IGNORE_PRIVATE_WARNING ["_unit", "_weapon", "_muzzle", "_mode", "_ammo", "_magazine", "_projectile", "_gunner"];
+//TRACE_8("firedEH:",_unit,_weapon,_muzzle,_mode,_ammo,_magazine,_projectile,_gunner);
+
+// Retrieve backblast values
+private _bbValues = [_weapon, _ammo, _magazine] call ace_overpressure_fnc_getOverPressureValues;
+
+_bbValues params ["_backblastAngle", "_backblastRange", "_backblastDamage", "_offset"];
+_backblastRange = _backblastRange * ace_overpressure_backblastDistanceCoefficient;
+
+//TRACE_4("cache",_backblastAngle,_backblastRange,_backblastDamage,_offset);
+
+if (_backblastDamage <= 0) exitWith {};
+
+private _direction = (vectorDir _projectile) vectorMultiply -1;
+private _position = (getPosASL _projectile) vectorAdd (_direction vectorMultiply _offset);
+
+// Damage to others
+private _affected = (ASLToAGL _position) nearEntities ["CAManBase", _backblastRange];
+
+// Let each client handle their own affected units
+["ace_overpressure", [_unit, _position, _direction, _weapon, _magazine, _ammo], _affected] call CBA_fnc_targetEvent;
+
+// Damage to the firer
+private _distance = 2 * ([_position, _direction, _backblastRange, _unit] call ace_overpressure_fnc_getDistance);
+
+//TRACE_1("Distance",_distance);
+
+if (_distance < _backblastRange) then {
+    //TRACE_2("",isDamageAllowed _unit,_unit getVariable [ARR_2(QEGVAR(medical,allowDamage),true)]);
+
+    // Skip damage if not allowed
+    if (isDamageAllowed _unit && {_unit getVariable ["ace_medical_allowDamage", true]}) then {
+        private _alpha = sqrt (1 - _distance / _backblastRange);
+        private _beta = sqrt 0.5;
+
+        private _damage = _alpha * _beta * _backblastDamage;
+        //TRACE_1("",_damage);
+
+        [_damage * 100] call BIS_fnc_bloodEffect;
+
+        if (missionNamespace getVariable ["ace_medical_enabled", false]) then {
+            if !(GVAR(ignoreArmorACE)) then {_damage = [_unit,  _damage, _damage, _unit, "", true] call EFUNC(main,receiveDamageACE);};
+            if (_damage > 0) then {[_unit, _damage, "body", "backblast", _unit] call ace_medical_fnc_addDamageToUnit};
```

Comment:
My suggestion is to add wound handlers for overpressure and fire via config. Then there is no need to do run `receiveDamageACE` here. tho it needs a new function to process the incoming damage then.

Might mean one more function then, but less hassle in keeping the overwritten ace functions up to date.